Feat: Airflow tags added to dag generation (#2558)
mozilla/bigquery-etl: docs/reference/airflow_tags.md

> # Airflow tags

Reviewer: ```suggestion
# Airflow Tags
```

## Why
Airflow tags enable DAGs to be filtered in the web ui view to reduce the number of DAGs shown to just those that you are interested in.

Additionally, their objective is to provide a little bit more information such as their impact to make it easier to understand the DAG and impact of failures when doing Airflow triage.

More information and the discussions can be found the the original [Airflow tags proposal](https://docs.google.com/document/d/1LqOCmadsC6kPTusqyczHha-mdG_DVsNWK4-a1-NHI_8).

> 
> ---
> 

Reviewer: ```suggestion

```

## Valid tags

### impact/tier tag

> We follow the same tiers as the ones defined for automation/treeherder. This is to maintain a level of consistency across different systems to ensure common language and understanding across teams. Valid tier tags include:

Reviewer: ```suggestion
We borrow the [tiering system](https://wiki.mozilla.org/Sheriffing/Job_Visibility_Policy#Overview_of_the_Job_Visibility_Tiers) used by our integration and testing sheriffs. This is to maintain a level of consistency across different systems to ensure common language and understanding across teams. Valid tier tags include:
```
